### PR TITLE
Upgrading test dependencies

### PR DESCRIPTION
**Pull request by Sam-Burns**
Uses PHPUnit 6 for development with PHP 7.  Bumps Behat version a bit.

### CODE REVIEW

File: integration/PhpSpec/Console/Prompter/QuestionTest.php [hunk 1]
```diff
@@ -3,12 +3,13 @@
 namespace integration\PhpSpec\Console\Prompter;
 
 use PhpSpec\Console\Prompter\Question;
+use PHPUnit\Framework\TestCase;
```
Comment (unfunco) [inline integration/PhpSpec/Console/Prompter/QuestionTest.php]: When the forward compatibility changes were first introduced in PHPUnit, the FQCN was `phpunit\framework\TestCase`, which will cause autoload issues, it was updated in either 5.5 or 5.6 to use the case you're using here, so this part shouldn't be changed until phpspec 4.0 unless the minimum version of PHPUnit is increased.

Comment (Sam-Burns) [inline integration/PhpSpec/Console/Prompter/QuestionTest.php]: Thanks for that.  I've upgraded the minimum PHPUnit version to `5.5`, so the build passes with `composer update --prefer lowest`.

**Comment by ciaranmcnulty** [comment]
@Sam-Burns can you target 4.0 branch please?

**Comment by Sam-Burns** [comment]
@ciaranmcnulty I wonder if you would consider removing the `bc-break` label from the PR, and consider it for merging into the `master` (PhpSpec v3) branch?

**Comment by ciaranmcnulty** [comment]
D'oh of course it still works with 5.x - my bad

**Pull request merged**